events/belgium/comsibal.yaml

```diff
@@ -78,3 +78,28 @@ events:
       - Viorel
       - Accords Perdus
     organisation: ComSibal
+
+  - name: ComSibal
+    links:
+      - "http://comsibal.be/en/"
+    start: 2023-02-18T19:00:00+01:00
+    end: 2023-02-18T23:55:00+01:00
```

Comment:
We already have this one in the Folkbalbende import, see `folkbalbende.yaml`.

Where did you get the times from?

Reply:
oh, should I move the other there too?

about the times: I just reused the last ones, because I didn't see any. Do you prefer just a date without time if there is none?